### PR TITLE
Add short names of the days of week for Russian localisation

### PR DESCRIPTION
**Pull request by andriipanasiuk**

### CODE REVIEW

File: klock/src/commonMain/kotlin/com/soywiz/klock/locale/ru.kt [hunk 1]
```diff
@@ -17,6 +17,11 @@ open class RussianKlockLocale : KlockLocale() {
 	override val daysOfWeek = listOf(
 		"воскресенье", "понедельник", "вторник", "среда", "четверг", "пятница", "суббота"
 	)
+
+	override val daysOfWeekShort = listOf(
+		"вск", "пн", "вт", "ср", "чт", "пт", "сб"
```
Comment (RezMike) [inline klock/src/commonMain/kotlin/com/soywiz/klock/locale/ru.kt:22]: I think it should be "вс", not "вск"

File: klock/src/commonTest/kotlin/com/soywiz/klock/locale/KlockLocaleTest.kt [hunk 2]
```diff
@@ -217,6 +217,9 @@ class KlockLocaleTest {
 
 		assertEquals("середа", DayOfWeek.Wednesday.localName(KlockLocale.ukrainian))
 		assertEquals("ср", DayOfWeek.Wednesday.localShortName(KlockLocale.ukrainian))
+
+		assertEquals("воскресенье", DayOfWeek.Sunday.localName(KlockLocale.russian))
+		assertEquals("вск", DayOfWeek.Sunday.localShortName(KlockLocale.russian))
```
Comment (RezMike) [inline klock/src/commonTest/kotlin/com/soywiz/klock/locale/KlockLocaleTest.kt:222]: The same here

File: klock/src/commonMain/kotlin/com/soywiz/klock/locale/ru.kt (hunk 1 above)
Comment (andriipanasiuk) [reply klock/src/commonMain/kotlin/com/soywiz/klock/locale/ru.kt:22]: Sure, thanks for your review.

**Comment by andriipanasiuk** [comment]
Will fix a short name for Sunday.

**Review by RezMike — APPROVED** [review]

**Comment by RezMike** [comment]
@andriipanasiuk Thank you for contribution!

**Pull request merged**